### PR TITLE
Clean up app icon url

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt [hunk 1]
```diff
@@ -18,15 +18,15 @@ object DefaultWallets {
     @JvmField
     val coinbaseWallet = Wallet(
         name = "Coinbase Wallet",
-        iconUrl = "https://play-lh.googleusercontent.com/wrgUujbq5kbn4Wd4tzyhQnxOXkjiGqq39N4zBvCHmxpIiKcZw_Pb065KTWWlnoejsg",
+        iconUrl = "https://wallet.coinbase.com/assets/images/favicon.ico",
```
Comment (bangtoven) [inline android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt:21]: not sure whether android image view can display .ico files

File: ios/CoinbaseWalletSDK/Client/Model/Wallet.swift [hunk 2]
```diff
@@ -38,15 +38,15 @@ public struct Wallet: Codable {
 extension Wallet {
     public static let coinbaseWallet = Wallet(
         name: "Coinbase Wallet",
-        iconUrl: URL(string: "https://is1-ssl.mzstatic.com/image/thumb/Purple122/v4/bd/2b/dc/bd2bdcac-44ff-0707-3ec1-fde5014a91a1/AppIcon-0-0-1x_U007emarketing-0-0-0-10-0-0-sRGB-0-0-0-GLES2_U002c0-512MB-85-220-0-0.png/100x100bb.jpg")!,
+        iconUrl: URL(string: "https://wallet.coinbase.com/assets/images/favicon.ico")!,
```
Comment (bangtoven) [inline ios/CoinbaseWalletSDK/Client/Model/Wallet.swift:41]: image

File: android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt (hunk 1 above)
Comment (bangtoven) [reply android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt:21]: we will be adding a new png file on assets/images path, but it's not hosted yet due to merge freeze

**Comment by bangtoven** [comment]
@vishnumad 🙏

File: android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt (hunk 1 above)
Comment (vishnumad) [reply android/walletsdk/src/main/java/com/coinbase/android/nativesdk/Wallet.kt:21]: This should work

**Review by vishnumad — APPROVED** [review]

**Comment by Gezegorz1** [comment]
Thank you

**Pull request merged**